vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -282,26 +282,36 @@ fire_and_forget WinRTHttpResource::PerformSendRequest(HttpRequestMessage &&reque
         reader.UnicodeEncoding(UnicodeEncoding::Utf8);
       }
 
-      // #9510 - 10mb limit on fetch
-      co_await reader.LoadAsync(10 * 1024 * 1024);
-      auto length = reader.UnconsumedBufferLength();
+      uint64_t segmentSize = 10 * 1024 * 1024;
+      auto contentLengthHeader = response.Content().Headers().ContentLength();
```

Comment:
note that this can be 0 if the content is of unknown size (it's being streamed), can you guard against that and fall back or do something different?

Reply:
I see there's an if just below : ) Maybe add a comment to clarify this?

Reply:
As discussed, `Content-Length` is actually not needed, so that code has been removed.